<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>527</width>
    <height>683</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add event data</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>46</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="minimumSize">
           <size>
            <width>436</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Surname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="surnameEdit">
          <property name="minimumSize">
           <size>
            <width>436</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Birth date and time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="birthdate">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="hrBox">
          <property name="maxCount">
           <number>24</number>
          </property>
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="minBox">
          <item>
           <property name="text">
            <string>00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>27</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>31</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>33</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>34</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>35</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>36</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>37</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>39</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>41</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>42</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>43</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>44</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>45</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>46</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>47</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>49</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>51</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>52</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>53</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>54</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>55</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>56</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>58</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>59</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sex:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sexBox">
          <item>
           <property name="text">
            <string>Prefer not to say</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>491</width>
          <height>91</height>
         </size>
        </property>
        <property name="title">
         <string>Birth location (if place of your birth is not present please try to search your nearest city)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>City:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="citySearch">
             <property name="minimumSize">
              <size>
               <width>431</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="cityList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Event:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="eventBox">
          <property name="minimumSize">
           <size>
            <width>341</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addTypeBtn">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Add type...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>491</width>
          <height>121</height>
         </size>
        </property>
        <property name="title">
         <string>Description / notes:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTextEdit" name="notesEdit">
           <property name="minimumSize">
            <size>
             <width>471</width>
             <height>91</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpBtn">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
